oauth2client/contrib/sql_alchemy.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
We should, but imports are messed up with that.

I'm adding `from __future__ import absolute_import`.